tests/coverage/mcdc/condition-limit.rs

```diff
@@ -0,0 +1,25 @@
+#![feature(coverage_attribute)]
+//@ edition: 2021
+//@ min-llvm-version: 18
+//@ compile-flags: -Zcoverage-options=mcdc
+//@ llvm-cov-flags: --show-branches=count --show-mcdc
```

Comment:
This needs a brief comment explaining what this test is testing for, as does the other test.